app/models/org.rb

```diff
@@ -33,38 +33,80 @@
 #
 
 class Org < ActiveRecord::Base
+  include ValidationMessages
+  include ValidationValues
   include GlobalHelpers
   include FlagShihTzu
   extend Dragonfly::Model::Validations
   validates_with OrgLinksValidator
 
+  LOGO_FORMATS = %w[jpeg png gif jpg bmp].freeze
+
   # Stores links as an JSON object: { org: [{"link":"www.example.com","text":"foo"}, ...] }
   # The links are validated against custom validator allocated at validators/template_links_validator.rb
   serialize :links, JSON
 
-  ##
-  # Associations
-#  belongs_to :organisation_type   # depricated, but cannot be removed until migration run
+
+  # ================
+  # = Associations =
+  # ================
+
   belongs_to :language
-  has_many :guidance_groups
+
+  belongs_to :region
+
+  has_many :guidance_groups, dependent: :destroy
+
   has_many :templates
+
   has_many :users
+
   has_many :annotations
 
   has_and_belongs_to_many :token_permission_types, join_table: "org_token_permissions", unique: true
 
   has_many :org_identifiers
+
   has_many :identifier_schemes, through: :org_identifiers
 
-  ##
-  # Validators
-  validates :name, presence: {message: _("can't be blank")}, uniqueness: {message: _("must be unique")}
+
+  # ===============
+  # = Validations =
+  # ===============
+
+  validates :name, presence: { message: PRESENCE_MESSAGE },
+                   uniqueness: { message: UNIQUENESS_MESSAGE }
+
+  validates :abbreviation, presence: { message: PRESENCE_MESSAGE }
+
+  validates :is_other, inclusion: { in: BOOLEAN_VALUES,
+                                    message: INCLUSION_MESSAGE }
+
+  validates :language, presence: { message: PRESENCE_MESSAGE }
+
+  validates :contact_email, presence: { message: PRESENCE_MESSAGE,
+                                        if: :feedback_enabled }
+
+  validates :org_type, presence: { message: PRESENCE_MESSAGE }
+
+  validates :feedback_enabled, inclusion: { in: BOOLEAN_VALUES,
+                                            message: INCLUSION_MESSAGE }
+
+  validates :feedback_email_subject, presence: { message: PRESENCE_MESSAGE,
```

Comment:
I think we can drop this now can't we? We removed it from the Org edit form. We would need to make sure that the user_mailer is no longer referencing the DB value though.

Reply:
I'm not sure what's required here. Should we wait for @xsrust to comment?